webdriver-spec.html

```diff
@@ -4624,636 +4646,562 @@
 </section> <!-- /Delete All Cookies -->
 </section> <!-- /Cookies -->
 
-<section>
-<h2>Interactions</h2>
-
-<p class=issue><strong>Warning:</strong>
- This section has not yet been redefined to match the <a>routing requests</a> model,
- and uses old concepts and definitions.
- Please do not rely on it yet.
+<section><!-- Actions -->
+  <h2>Interactions</h2>
+  The Actions API gives you a mechanism to do atomic interactions by sending over
+  a JSON object that can then be used to <a>dispatch commands</a>.
+  The commands are then <a data-lt=command-processed>processed</a> by the user agent,
+  allowing complex interactions to be created.
+  <aside class=example>
+    If a person wanted to emulate a user doing <code>click + drag + drop</code>
+    they would use the following section to generate an object that is
+    then sent over to the <a>remote end</a>
+  </aside>
+
+  <p class='note'>
+    An <a>activation trigger</a> generated by the WebDriver API User SHOULD be
+    indistinguishable from those generated by a real user interacting with the
+    browser. In particular, the dispatched events will have the
+    <a>isTrusted</a> attribute set to true. The most robust way to dispatch
+    these events is by creating them in the browser implementation itself.
+    Sending OS-specific input messages to the browser's window has the
+    disadvantage that the browser being automated may not be properly isolated
+    from a user accidentally modifying input source state; use of an OS-level
+    accessibility API has the disadvantage that the browser's window must be
+    focused, and as a result, multiple tests cannot run in parallel.
```

Comment:
“Multiple tests in parallel” feels like the wrong vernacular to describe what WebDriver does, but I understand what you’re trying to say.